pkg/config/json_test.go

```diff
@@ -274,7 +274,7 @@ var jsonStabilityTests = map[string]jsonStabilityTest{
 			return blk
 		},
 
-		skipFields:   []string{"DevName"},
+		skipFields:   []string{"DevName", "Uri"},
```

Comment:
I'm surprised by the `Uri` capitalization, it is `URI` in the struct, do we need to skip this `Uri` field?

Reply:
Maybe you were looking at an old version? I can see it is actually URI in the code.

Reply:
Confused me when looking again, but the first commit adds `Uri`, the second commit changes this to `URI`

Reply:
Ah i see. At first it was Uri, then lint complained but it seems I didn't fix all commits 👍

Reply:
Updated, sorry I didn't see it before. Thanks!!